examples/options/src/options/full_tracking_input_options.cpp

```diff
@@ -10,9 +10,6 @@
 
 traccc::full_tracking_input_config::full_tracking_input_config(
     po::options_description& desc) {
-
-    desc.add_options()("detector_file", po::value<std::string>()->required(),
-                       "specify detector file");
```

Comment:
I think line 19 also need to be removed?

Reply:
Good point, done.